docs/admin/workloads/inference-overview.md

```diff
@@ -1,53 +1,52 @@
+* Inference is implemented as a Kubernetes *Deployment* object with a defined number of replicas. The replicas are load-balanced by Kubernetes so adding more replicas will improve the overall throughput of the system.
 
 * Multiple replicas will appear in Run:ai as a single Inference workload. The workload will appear in all Run:ai dashboards and views as well as the Command-line interface.
 
-* Inference workloads can be submitted via Run:ai [user interface](../admin-ui-setup/deployments.md) as well as [Run:ai API](../../developer/cluster-api/workload-overview-dev.md). Internally, spawning an Inference workload also creates a Kubernetes _Service_. The service is an end-point to which clients can connect. 
+* Inference workloads can be submitted via Run:ai [user interface](../admin-ui-setup/deployments.md) as well as [Run:ai API](../../developer/cluster-api/workload-overview-dev.md). Internally, spawning an Inference workload also creates a Kubernetes *Service*. The service is an end-point to which clients can connect.
 
 ## Auto Scaling
 
-To withstand SLA, Inference workloads are typically set with _auto scaling_. Auto-scaling is the ability to add more computing power (Kubernetes pods) when the load increases and shrink allocated resources when the system is idle.
+To withstand SLA, Inference workloads are typically set with *auto scaling*. Auto-scaling is the ability to add more computing power (Kubernetes pods) when the load increases and shrink allocated resources when the system is idle.
 
 There are a number of ways to trigger auto-scaling. Run:ai supports the following:
 
 | Metric          | Units        |   Run:ai name   |
 |-----------------|--------------|-----------------|
-| GPU Utilization |   %          | gpu-utilization |
-| CPU Utilization |   %          | cpu-utilization |
-| Latency         | milliseconds | latency         |
 | Throughput      | requests/second | throughput |
-| Concurrency     |              |    concurrency  | 
-| Custom metric   |              |    custom       |
+| Concurrency     |              |    concurrency  |
 
 The Minimum and Maximum number of replicas can be configured as part of the autoscaling configuration.
 
-Auto Scaling also supports a scale to zero policy with _Throughput_ and _Concurrency_ metrics, meaning that given enough time under the target threshold, the number of replicas will be scaled down to 0.
-This has the benefit of conserving resources at the risk of a delay from "cold starting" the model when traffic resumes. 
-
+Auto Scaling also supports a scale to zero policy with *Throughput* and *Concurrency* metrics, meaning that given enough time under the target threshold, the number of replicas will be scaled down to 0.
+This has the benefit of conserving resources at the risk of a delay from "cold starting" the model when traffic resumes.
 
 ## See Also
 
```

Comment:
is quickstart-inference.md up to date? with asset-based instructions?
is deployments.md up to date? should not be called deployments anymore

Reply:
@shayasoolin good point about the quickstart. Will try to look into it. As far as deployments, that is something that needs to be changed. It currently is on my list for changes to the docs in this version. It is a little more complicated than a "find / replace" though.

Reply:
@liohill the quickstart for inference needs to have a review to make sure it is still accurate.